dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScopeTest.kt

```diff
@@ -99,6 +100,9 @@ internal class RumActionScopeTest {
     @Mock
     lateinit var mockRumEventSourceProvider: RumEventSourceProvider
 
+    @BoolForgery
+    var fakeTrackFrustrations: Boolean = true
```

Comment:
seems this is the only thing remaining to address in the test, it should be:

```suggestion
    lateinit var fakeTrackFrustrations: Boolean
```

same for other similar occurrences in other files.

Reply:
The change makes sense, but `lateinit` isn't allowed on primitive type as far as I could see. 
(I'm still not the most experienced kotlin dev)

Reply:
yes, you are right :) I missed that